deploy.cfg

```diff
@@ -45,6 +46,17 @@ collective.roster = git git://github.com/collective/collective.roster.git pushur
 [instance]
 effective-user = plone
 
+event-log-custom =
+    %import raven.contrib.zope
+    <logfile>
+      path ${buildout:directory}/var/log/instance.log
+      level INFO
+    </logfile>
+    <sentry>
+      dsn https://75b1a7aebce7404c8a467b25b608a9e5:439368748d494f07b9e86b09419dc753@sentry.io/128640
```

Comment:
`user:password` well done!

Reply:
@hvelarde we see how overworked and tiered I am. 

Well a write only user in this case is like a git deploy key for public readable repos is not critical. 
What makes me more sceptical about my work, is that I pushed a config for a superseeded working branch, that contains a config to a tool, that we do not use and was for my development only.

Reply:
I feel you pain, believe me.